### ISSUE TITLE
`make build` fails on workstation

### ISSUE THREAD

**Issue by ramasesh** [google/uv-metrics#7]
On workstation, the command `make build` fails when trying to create an VirtualEnv with `python3.6` installation.  The full output is as follows:
```
rm -rf env
rm -rf .tox
rm -rf .coverage
find . -name \*.pyc -type f -delete
find . -name \*.test.db -type f -delete
find . -depth -name __pycache__ -type d -exec rm -rf {} \;
rm -rf dist *.egg* build
rm -rf env
virtualenv -p python3.6 env
RuntimeError: failed to find interpreter for Builtin discoverer of python_spec='python3.6'
make: *** [Makefile:43: install] Error 1
```
So it's the command `virtualenv -p python3.6 env` which is failing, and this same failure happens when I try to run that command in shell.  

As @ajslone suggested, changing the version to `python3`, i.e. using the command `virtualenv -p python3 env` by changing the variable `PYTHON_BIN=python3` in the makefile does work.

**Comment by sritchie** [comment]
I'm okay with this change, since we now have CI testing on all python versions!

**Comment by sritchie** [comment]
Closed by #18 .
